tidy-data/01-Reshape-Data/01-Reshape-Data.Rmd

```diff
@@ -621,10 +657,13 @@ babynames %>%
 babynames %>%
   group_by(year, sex) %>% 
   summarise(total = sum(n)) %>%
```

Comment:
I'm guessing previous primers covered this and not `count` 😇

Reply:
I'm going to leave this one as is to not be changing multiple aspects at this time. Happy to do a review of count/summarise of all primers if we decide that's important.